image_provisioner/playbooks/roles/repo-install/templates/etc/yum.repos.d/all.repo.j2

```diff
@@ -53,15 +53,6 @@ enabled=1
 gpgcheck=0
 sslverify=0
 
-# This htb repo gets us openvswitch-2.6, and is required until the htb bits are promoted to the above GA channel (GA is openvswitch-2.5 for now).
```

Comment:
do we need to remove this repo, in case we need other packages from it in future ?